### PR TITLE
Add a function to get course history

### PR DESCRIPTION
**Pull request by areyoualex**
Although get_schedule is sufficient for getting the current year's schedule, it would be nice to get a user's past grades. This PR adds a StudentVue.get_course_history function, which for now, returns a dictionary of format:
```python
{
  '6': [ # Grade level
    ['English 6A', 'Math 6A', ...], # 1st semester
    ['English 6B', 'Math 6B', ...] # 2nd semester
  ],
  ...
}
```
Each of the courses is of type studentvue.models.Course, which has the name, letter grade, credits attempted, and credits completed of a course, as well as if its an AP course.

Also, I added a unit testing folder to test that this function works. (If you decide to test a different way, that's fine, you can remove it.)

### CODE REVIEW

**Comment by areyoualex** [comment]
If you consider merging this, should it have a different return type? Perhaps a data model to more easily get a list of courses, like so:
```python
sv = StudentVue(user, pass, domain)

course_history = sv.get_course_history

print(course_history.get_semester(6,'F'))
# prints array of Course for that semester
```

File: studentvue/models.py [hunk 1]
```diff
@@ -39,3 +39,14 @@ def __init__(self, name, class_name, date, assignment_id, grading_period, org_ye
         super().__init__(name, class_name, date, assignment_id, grading_period, org_year_id)
         self.score = score
         self.max_score = max_score
+
+class Course:
+    def __init__(self, name, grade, creditsAttempted, creditsCompleted, isAP):
```
Comment (kaijchang) [inline studentvue/models.py:44]: Can you switch to snake_case to keep consistency with the rest of the library?

File: studentvue/studentvue.py [hunk 2]
```diff
@@ -237,6 +237,33 @@ def get_class_info(self, class_):
 
         return self._parse_grade_book_class_page(grade_book_class_page, class_.name)
 
+    def get_course_history(self):
```
Comment (kaijchang) [inline studentvue/studentvue.py:240]: I can look into it more later, but for me, it's only getting the first semester.
```
{
   '9':[
      [
         CCSS Geometry B (GEOC150B) with grade of A,
         Chinese 3B H (CHIN253B) with grade of A,
         College/Career (COCA150) with grade of A,
         English 1B (ENGL151B) with grade of A,
         NGSS Biology B (BION150B) with grade of A,
         PE 1B (PEPE151B) with grade of A,
         World History B (WHST150B) with grade of A
      ]
   ],
   '10':[
      [
         AP Chinese B (CHIN350B) with grade of A,
         AP Euro Hist B (EUHI350B) with grade of A,
         CCSS Algebra 2B (ALGC152B) with grade of A,
         Chemistry B H (CHEM250B) with grade of A,
         Comp Prg B (PROG150B) with grade of A,
         English 2B (ENGL152B) with grade of A,
         Fit for Life 2B (FITL152B) with grade of A
      ]
   ]
}
```

File: studentvue/tests/__init__.py [hunk 3]
```diff
@@ -0,0 +1 @@
+from .test_course_history import TestCourseHistory
```
Comment (kaijchang) [inline studentvue/tests/__init__.py]: When I run `python3 -m unittest`, it's detecting this and running the test the second time. Removing the import fixes this.

File: studentvue/tests/test_course_history.py [hunk 4]
```diff
@@ -0,0 +1,17 @@
+from unittest import TestCase
```
Comment (kaijchang) [inline studentvue/tests/test_course_history.py:1]: This is fine for now, but my plan for tests is to abstract the parsing of the html / data from the fetching of the data, hence the static `_parse...` methods. This way, there are no credentials involved, and you can just pass in some mocked up html and see if it spits out the right result, so you can also test edge cases beyond what's in your own account.

**Review by kaijchang — CHANGES_REQUESTED** [review]

File: studentvue/studentvue.py (hunk 2 above)
Comment (areyoualex) [reply studentvue/studentvue.py:240]: Silly me, it was an indentation error I didn't catch. It works fine now (at least for me)
```python
{
  '6': [
    [Homeroom(HMRM050) with grade of N, Japanese 1(JAPN051) with grade of A,
      Language Arts 6 Honors(LANG256) with grade of A,
      Math 6 Honors(MATH256) with grade of A,
      Physical Education 6(PEPE056) with grade of B,
      Science 6 Honors(SCIE256) with grade of A,
      Social Studies 6 Honors(SOCS256) with grade of A],
    [Homeroom(HMRM050) with grade of N, Japanese 1(JAPN051) with grade of A,
      Language Arts 6 Honors(LANG256) with grade of B,
      Math 6 Honors(MATH256) with grade of A,
      Physical Education 6(PEPE056) with grade of A,
      Science 6 Honors(SCIE256) with grade of A,
      Social Studies 6 Honors(SOCS256) with grade of A]
  ],
  '7': [
    [CCSS Math 7(CCMA057) with grade of A,
      Homeroom(HMRM050) with grade of N,
      Japanese 2(JAPN052) with grade of A,
      Language Arts 7 Honors(LANG257) with grade of A,
      Physical Education 7(PEPE057) with grade of A,
      Science 7 Honors(SCIE257) with grade of A,
      Social Studies 7 Honors(SOCS257) with grade of B],
    [CCSS Math 7(CCMA057) with grade of A,
      Homeroom(HMRM050) with grade of N,
      Japanese 2(JAPN052) with grade of A,
      Language Arts 7 Honors(LANG257) with grade of A,
      Physical Education 7(PEPE057) with grade of B,
      Science 7 Honors(SCIE257) with grade of A,
      Social Studies 7 Honors(SOCS257) with grade of A]
  ],
...
]
```
